src/ray/core_worker/core_worker.h

```diff
@@ -324,6 +324,10 @@ class CoreWorker : public rpc::CoreWorkerServiceHandler {
 
   const JobID &GetCurrentJobId() const { return worker_context_.GetCurrentJobID(); }
 
+  ClientID GetCurrentNodeId() const {
```

Comment:
const return?

Reply:
The return value is a copy not a reference. So, the `const` is not necessary.